src/twisted/trial/_asynctest.py

```diff
@@ -58,6 +58,12 @@ def __init__(self, methodName="runTest"):
         """
         super().__init__(methodName)
 
+        # Acquire and store real reactor so that it does not interfere with any
+        # patching to the reactor done by the user.
+        from twisted.internet import reactor
+
+        self.__reactor = reactor
```

Comment:
Twisted doesn't use the double underscore prefix.  Private attributes are just normal private attributes.

Reply:
I agree in general, but in this particular case I think it's worth considering double prefix. 

TestCase class is by the users, so any attribute we use may clash with with anything users sets on the instance. It is especially true for `_reactor`, because users who write tests with some kind of mock reactor are likely to store it into test case attributes with similar name. I've just looked into the project I maintain - Buildbot - and there's already cases where `_reactor` is set on the test case.

Reply:
Just pick a different name, then

Reply:
Renamed to `_twistedPrivateReactor`, let's hope the new name is not too verbose.

I think the rest of attributes should have similar names.